src/modules/blocks/ticket/container-header/quantity/template.js

```diff
@@ -27,16 +27,12 @@ const TicketContainerHeaderDescription = ( {
 } ) => {
 	const total = isShared ? sharedCapacity : capacity;
 
-	const getLabel = () => (
-		isUnlimited
-			? sprintf( __( '%d sold', 'event-tickets' ), sold )
-			: sprintf( __( '%d of %d sold', 'event-tickets' ), sold, total )
-	);
+	const getLabel = () => sprintf( __( '%d sold', 'event-tickets' ), sold );
 
 	const getQuantityBar = () => (
 		isUnlimited
 			? (
-				<span className="tribe-editor__ticket__container-header-quantity-unlimited">
+				<span className="tribe-editor__ticket__container-header-quantity-unlimited tribe-editor__ticket__container-header-label">
```

Comment:
Should this match `tribe-editor__ticket__container-header-quantity-label` - it seems they are the same styling (but maybe I missed something in the video?)

Reply:
They don't since this is just for the unlimited text